wordpress-mobile/WordPress-iOS
# Fix broken add site button

Currently in `develop` the add site button on the blog list (the `+` button) does nothing. This is due to a mistake I made in #9688.

Thanks to @ScoutHarris for reporting it.

## To test:
- Ensure that adding a new site still works in these situations:
  - When tapping the `+` button in the blog list
  - When tapping the write button with an account with no blogs
  - When tapping the blue button on the my sites tab of an account with no blogs
- Ensure that tapping the `+` button when all blogs have been hidden still switches to edit mode
- Finally, ensure all these work on both iPad and iPhone
